wordle-app/pages/WordEditor.vue

```diff
@@ -0,0 +1,245 @@
+<template>
+  <v-container style="max-width: 800px">
+    <v-toolbar flat class="mb-3">
+      <v-toolbar-title>Word Editor</v-toolbar-title>
+      <v-spacer></v-spacer>
+      <v-switch
+        label="Display common words only"
+        color="secondary"
+        @change="fetchWords"
+        class="d-flex justify-center"
+        v-model="isCommonFilter"
+      />
+      <v-btn
+        v-if="isLogged"
+        color="primary"
+        prepend-icon="mdi-plus-box"
+        variant="flat"
+        class="ml-5"
+        @click="showDialog = true"
+        >Add Word
+      </v-btn>
+    </v-toolbar>
+
+    <v-data-table-server
+      v-model:items-per-page="options.itemsPerPage"
+      :headers="headers"
+      :items="words"
+      :items-length="totalWords"
+      :options.sync="options"
+      @update:options="fetchWords"
+      @update:page="pageNumber"
+      @update:items-per-page="itemsPerPage"
+      :loading="isLoading"
+      :items-per-page-options="[10, 25, 50, 100]"
```

Comment:
great job on the extra credit here!

Reply:
It was more of a bug fix because v-data-table implements this feature automatically and it has an option to select all words from the list. It was buggy so I had to remove it by adding this line.